SimTracker/SiPhase2Digitizer/test/PixelTestBeamValidation.cc

```diff
@@ -0,0 +1,738 @@
+//
```

Comment:
much like other tracker validation code, this should move into a Validation or DQM package. I am not very keen on adding yet more validation code in the wrong location.

Reply:
Again, let me refer to thee [previous comment](https://github.com/cms-sw/cmssw/pull/29018#discussion_r387023499). This validator will be only used during the development of this simulation code, to cross check with real data from test beams. Once the code is finished, or at least, the main bulk of the simulation is validate, and we are sure the response of the sensor is realistic, I don't think that class will be used anymore, as test beam data won't be used afterwards, but collision data. The validator is not intended to be used in any DQM sequence neither, just locally, for tests and cross-checks.
Therefore, I would prefer to keep it in the test area, and it will be removed whenever it is not needed anymore, or if eventually we decided it is worth it to included it in any sequence, then we can move it to the DQM package.

Reply:
@kpedro88 

> much like other tracker validation code

Off-topic, but just to be sure I understand what you mean, please name an example.

Reply:
In #28432 a bunch of validation modules were moved out of `SimTracker/SiPhase2Digitizer/test` into a dedicated DQM package (which is where such code belongs).

Reply:
so, it's no longer the case, isn't it?

Reply:
We should not regress by adding more validation code in `SimTracker/SiPhase2Digitizer/test`.

Reply:
no, but your comment was implying this was not yet clean, while in fact it is.

Reply:
Sorry, I should have specified that this was a problem in the past that I would prefer to avoid repeating.